src/server/multiplayer/MultiplayerServerState.js

```diff
@@ -110,24 +133,35 @@ export default class MultiplayerServerState {
 
   // Make a player join the given opponent's hosted game with the given deck.
   // Returns the game joined.
-  joinGame = (clientID, opponentID, deck) => {
+  joinGame = (clientID, opponentID, deck, gameProps = {}) => {
     const waitingPlayer = find(this.state.waitingPlayers, { id: opponentID });
+    const gameId = generateID();
+
     const game = {
-      id: opponentID,
-      name: waitingPlayer.name,
+      id: gameId,
+      name: `Casual#${waitingPlayer.name}`,
```

Comment:
What's the point of this?

Reply:
I guess we don't really need this but let's keep it for now.